### PR TITLE
Shows a warning for people tempted to switch from the Obsolete to the new MediaPicker(v3)

### PR DESCRIPTION
**Pull request by nul800sebastiaan**
This is in relation to #10308, please read that first.

Which looks like this: 

image

### CODE REVIEW

**Comment by nul800sebastiaan** [comment]
Relates to: #10308

File: src/Umbraco.Web/PropertyEditors/MediaPickerConfiguration.cs [hunk 1]
```diff
@@ -8,6 +8,9 @@ namespace Umbraco.Web.PropertyEditors
     /// </summary>
     public class MediaPickerConfiguration : IIgnoreUserStartNodesConfig
     {
+        [ConfigurationField("notice", "You can NOT change the property editor", "obsoletemediapickernotice")]
+        public bool Notice { get; set; }
+
```
Comment (bjarnef) [inline src/Umbraco.Web/PropertyEditors/MediaPickerConfiguration.cs:13]: @nul800sebastiaan I think this should be defined as `string`.

Comment (nul800sebastiaan) [reply src/Umbraco.Web/PropertyEditors/MediaPickerConfiguration.cs:13]: sure but it doesn't really matter, it's not a REAL prevalue, just a hack to show a warning for now while we think of a betters solution. 😄

**Comment by skttl** [comment]
I see the good intention, but isn't this the case for most data editors already?

**Comment by nathanwoulfe** [comment]
I think it's valuable since previous obsolete editors can be swapped out - ContentPicker for ContentPicker2, for example, works fine. 
Probably a good pattern to adopt for all obsolete/superseded patterns, so users can make changes knowing they're not going to lose 1500 images from their site...

**Comment by nul800sebastiaan** [comment]
> I see the good intention, but isn't this the case for most data editors already?

@skttl yes of course, this is just a first step since we already KNOW that people will try it and get burned: https://github.com/umbraco/Umbraco-CMS/issues/10308

We need to expand this somehow but we don't have the bandwith before release.

**Pull request merged**